src/server/server_family.cc

```diff
@@ -1172,18 +1172,6 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
     }
   }
 
-  const auto& tc = m.coordinator_stats.tx_type_cnt;
-  AppendMetricHeader("transaction_types_total", "Transaction counts by their types",
-                     MetricType::COUNTER, &resp->body());
-
-  const char* kTxTypeNames[ServerState::NUM_TX_TYPES] = {"global", "normal", "quick", "inline"};
```

Comment:
why deleting this?

Reply:
Because we don't have neither quick nor inline. Those are not disjoint types along with global or normal. I can add normal_cnt and global_cnt metrics

Reply:
what's the equivalent of quick now?

Reply:
there is none, there is a per-shard counter of immediate runs